### PR TITLE
tpm_with_bitlocker: New case to test virtual TPM device by BitLocker

### PR DESCRIPTION
**Pull request by YongxueHong**
New case to test virtual TPM device by BitLocker inside window guest.

ID: 1861614
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2673

**Comment by YongxueHong** [comment]
@qcheng-redhat  @vivianQizhu 
Please help me to review, thanks.
CC @CongLi

File: qemu/tests/cfg/tpm_with_bitlocker.cfg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    kill_vm = yes
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    tpms = tpm0
+    tpm_backend_tpm0 = emulator
+    tpm_model_tpm0 = tpm-crb
+    tpm_version_tpm0 = 2.0
+    Win2019, Win2008, Win2016, Win2012:
+        cmd_install_bitlocker = 'powershell -command "Install-WindowsFeature -Name BitLocker -Restart"'
```
Comment (qcheng-redhat) [inline qemu/tests/cfg/tpm_with_bitlocker.cfg:16]: We could remove -Restart here, with this parameter here, I got error:
04:21:33 ERROR| ERROR 1-Host_RHEL.m8.u3.product_av.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.tpm_with_bitlocker.tpm_with_bitlocker.q35 -> ShellStatusError: Could not get exit status of command: 'powershell -command "Install-WindowsFeature -Name BitLocker -Restart"'    (output: '\nSuccess Restart Needed Exit Code      Feature Result                               \n------- -------------- ---------      --------------                               \nTrue    Yes            SuccessRest... {BitLocker Drive Encryption, Enhanced Stor...\nWARNING: You must restart this server to finish the installation process.\n\n\n\n')
04:21:33 INFO | 

And we have reboot vm operation just after bitlocker installation in .py already. 
        session.cmd(cmd_install_bitlocker, 360)
        session = vm.reboot(session, timeout=480)

Comment (YongxueHong) [reply qemu/tests/cfg/tpm_with_bitlocker.cfg]: Thanks, have updated.

**Comment by YongxueHong** [comment]
@qcheng-redhat
Please help me to review, thanks.

File: qemu/tests/cfg/tpm_with_bitlocker.cfg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    tpms = tpm0
+    tpm_type_tpm0 = emulator
+    tpm_model_tpm0 = tpm-crb
+    tpm_version_tpm0 = 2.0
+    Win2019, Win2008, Win2016, Win2012:
+        cmd_install_bitlocker = 'powershell -command "Install-WindowsFeature -Name BitLocker"'
+    cmd_bdehdcfg = 'bdehdcfg -target c: shrink -newdriveletter s: -size 300 -quiet'
```
Comment (qcheng-redhat) [inline qemu/tests/cfg/tpm_with_bitlocker.cfg:19]: Please help update -size 300 to -size 606. Thanks.

Comment (YongxueHong) [reply qemu/tests/cfg/tpm_with_bitlocker.cfg]: Got it , have updated.

**Comment by qcheng-redhat** [comment]
LGTM. Thanks. 

python3 ConfigTest.py --platform=x86_64 --guestname=Win2019,Win2016 --testcase=tpm_with_bitlocker --machine=q35 --firmware=ovmf --clone=no

(1/2) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.tpm_with_bitlocker.q35: PASS (575.70 s)
 (2/2) Host_RHEL.m8.u3.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.tpm_with_bitlocker.q35: PASS (544.02 s)

There vtpm bug on Win10 guest.

**Pull request merged**